src/View/Helper/FormHelper.php

```diff
@@ -260,30 +260,30 @@ public function widgetRegistry(WidgetRegistry $instance = null, $widgets = [])
      */
     protected function _addDefaultContextProviders()
     {
-        $this->addContextProvider('orm', function ($request, $data) {
+        $this->addContextProvider('orm', function ($request, $data, $requestType) {
```

Comment:
Method doesn't have a `$requestType` var, you probably meant to use `$this->requestType`